hack/gitea/ingress.yaml.tmpl

```diff
@@ -2,6 +2,70 @@
 ---
 apiVersion: networking.k8s.io/v1
 kind: Ingress
+metadata:
+  name: my-gitea-path-oci-root
+  namespace: gitea
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 512m
```

Comment:
Only thought or question on this is 512m big enough if I am pushing and pulling 1GB+ images?

Reply:
@nabuskey I think you chose this limit? I agree that many images may have layers > 1GB.

Reply:
Yes this is a possibility. 512m was chosen for some reason I cannot remember... We can bump it up I think.